src/styles.scss

```diff
@@ -59,7 +59,7 @@ input {
   justify-content: space-between;
 }
 
-button {
+.btn {
   position: relative;
   display: flex;
   padding: 0.5rem 1rem;
```

Comment:
This broke save button on article card can you please fix that

Reply:
Done